packages/typespec-vscode/src/vscode-cmd/emit-code/emit-code.ts

```diff
@@ -153,60 +156,82 @@ async function doEmit(mainTspFile: string, emitter: Emitter) {
       cancellable: false,
     },
     async () => {
-      const packageQuickPickItems = [];
-      if (action === InstallAction.Upgrade || action === InstallAction.Install) {
-        const minimumRequisites = emitter.requisites
-          ? ` Minimum requisites: install ${emitter.requisites?.join(", ")}`
-          : "";
-        let packageFullName = emitter.package;
-        if (version) {
-          packageFullName = `${emitter.package}@${version}`;
+      const installCalculationPromises = emitters.map(async (emitter) => {
+        const packageQuickPickItems: any[] = [];
```

Comment:
is there no type fo this?

Reply:
Hello @timotheeguerin This is one time use, we don't need to define an type for it. Typescript will recognize it.

Reply:
Im talking about any, you telling it's anything now so there is no validation below when you push to the array

Reply:
Thanks @timotheeguerin We should not use any, and then typescript will do the type validation when push item to the array.